modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/viewmodel/AdvancedSettingsViewModel.kt

```diff
@@ -68,13 +61,12 @@ class AdvancedSettingsViewModel
     }
 
     data class State(
-        val refreshPodcastsAutomatically: Boolean,
         val backgroundSyncOnMeteredState: BackgroundSyncOnMeteredState
     ) {
 
         data class BackgroundSyncOnMeteredState(
-            @StringRes val summary: Int,
             val isChecked: Boolean = true,
+            val isEnabled: Boolean = true,
             val onCheckedChange: (Boolean) -> Unit,
         )
     }
```

Comment:
What you have here is fine, but I would probably collapse the `BackgroundSyncOnMeteredState` fields directly into the `State` object since `State` currently only has that single field:

``` kotlin
    data class State(
        val isChecked: Boolean = true,
        val isEnabled: Boolean = true,
        val onCheckedChange: (Boolean) -> Unit,
    )
```

Reply:
True, my only thought with collapsing it inside is that if we decide to create or move other options in here and then it's not the only row state, then we'd have to break it out again. 🤷